### PR TITLE
[Docs](bangc-ops):Fixed doc problems

### PR DESCRIPTION
**Pull request by GraceGuanH**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

## 3. Test Report

If you want to know how to do operator testing, you can see [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).

#### 3.1.1 Accuracy Acceptance Standard

For static threshold standard details, see: [MLU-OPS Accuracy Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Accuracy-Acceptance-Standard.md).

- [ ] diff1: diff1 <= 3e-3
- [ ] diff2: diff2 <= 3e-3

#### 3.1.2 Operator Scheme checklist

|     No.        |                 Details              |            Check Results             |
|----------------|--------------------------------------|--------------------------------------|
|        1       |Supported hardware                    |             MLU370<br>MLU590         |
|        2       |Job types                             |          block <br> U1 <br> U4       |
|        3       |Layouts                               |          NHWC 、NCHW、ARRAY etc      |
|        4       |Whether multi-dimensions are supported|                                      |
|        5       |Whether element zero is supported     |                                      |
|        6       |Data type(half/float)                 |           half / float etc           |
|        7       |Whether there is size limit           |                                      |

#### 3.1.3 New Feature Test

If you have checked the following items, please tick the relevant box.

- [ ] Data type test
- [ ] Multi-dimensional tensor test
- [ ] Layout test
- [ ] Different size/integer remainder end segment/alignment misalignment test
- [ ] Zero dimensional tensor test/zero element test
- [ ] stability test
- [ ] Multiple platform test
- [ ] Gen_case module test
- [ ] Nan/INF tests 
- [ ] Bug fix tests
- [ ] For memory leak check details, see[GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For code coverage check details, see: [GTest-User-Guide-zh](https://github.com/Cambricon/mlu-ops/blob/master/docs/GTest-User-Guide-zh.md).
- [ ] For I/O calculation efficiency check details, see: [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md).

#### 3.1.4 Parameter Check

When a new operator is submitted, the test points are given and the test results are stated.

|                   Test Point                    | Acceptance Standard | Test Result (Error Message) |
| ----------------------------------------------- | --------------------| --------------------------- |
| Whether it conforms to the operator restriction |     Normal error    |                             |
| Whether illegal parameters are passed           |     Normal error    |                             |

### 3.2 Accuracy Test

For the cases used in the New Feature Test section, the features and the number of cases are recorded here. When multiple operations are tested, multiple tables are needed to include details of these operations.

Operation:

|Test Point           | Description                      | Quantity |  Comment |
|----------           |----------------------------------|----------|  --------|
|Data type test       |half/float/int8                   |          |          |
|Mult-tensor test     |Supports 1-8 dims                 |          |          |
|Layout test          |Supports NCHW/NHWC                |          |          |
|Zero element test    |Whether to support this test      |          |          |
|Stability test       |--gtest_repeat=NUM<br>--thread=NUM|          |          |
|Mult-platform test   |MLU370/MLU590                     |          |          |
|Nan/INF test         |Whether to support this test      |          |          |
|Memory leak check    |Test result                       |          |          |
|Code coverage check  |Test result                       |          |          |

### 3.3 Performance Test

See [MLU-OPS Performance Acceptance Standard](https://github.com/Cambricon/mlu-ops/blob/master/docs/MLU-OPS-Performance-Acceptance-Standard.md) for details.

Platform：MLU370

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|-----|----|----|----|-----|
|op_name|    |    |     |    |    |    |     |
|op_name|    |    |     |    |    |    |     |

Platform：MLU590

|Operation|Mlu_hardware_time(us)|Mlu_interface_time(us)|Mlu_io_efficiency|Mlu_compute_efficiency|Mlu_workwpace_size(Bytes)|Data_type|Shape|
|-------|----|----|----|----|----|----|-----|
|op_name|    |    |    |    |    |    |     |
|op_name|    |    |    |    |    |    |     |

### 3.4 Summary Analysis

Please give a brief overview here, if you want to note and summarize the content.

### CODE REVIEW

**Review by DanieeelLiu — APPROVED** [review]

File: bangc-ops/mlu_op.h [hunk 1]
```diff
@@ -2246,7 +2240,7 @@ mluOpGetGenerateProposalsV2WorkspaceSize(mluOpHandle_t handle,
  *   - rpn_rois_batch_size: int32
  *
  * @par Data Layout
- * - The supported data layout of \b input, \b output,
+ * - The supported data layouts of \b input, \b output,
  *   \b output_size are as follows:
  *
  * - input tensor: \p MLUOP_LAYOUT_ARRAY
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:2246]: 删除上面的空行，且下面内容空两格

File: bangc-ops/mlu_op.h [hunk 2]
```diff
@@ -3014,17 +3007,17 @@ mluOpSqrt(mluOpHandle_t handle,
  * queues in the sqrt backward operation. For detailed information, see
  * ::mluOpHandle_t.
  * @param[in] y_desc
- * The descriptor of the tensors. For detailed information, see
+ * The descriptor of the tensor. For detailed information, see
```
Comment (tudejiang79) [inline bangc-ops/mlu_op.h:3010]:
```suggestion
 * The descriptor of the tensor \b y. For detailed information, see
```

Comment (tudejiang79) [reply bangc-ops/mlu_op.h:3010]: 其它的雷同

**Review by tudejiang79 — APPROVED** [review]

**Review by tudejiang79 — APPROVED** [review]

File: bangc-ops/mlu_op.h (hunk 1 above)
Comment (duzekunKTH) [reply bangc-ops/mlu_op.h:2246]:
```suggestion
 *   - input tensor: \p MLUOP_LAYOUT_ARRAY
```

File: bangc-ops/mlu_op.h [hunk 3]
```diff
@@ -2455,35 +2449,35 @@ mluOpPolyNms(mluOpHandle_t handle,
...
+ * The descriptor of the max_sizes tensor. The maximum size of generated
  * prior boxes.
  * @param[in] max_sizes
  * Pointer to the MLU memory that stores the max_sizes tensor.
  * @param[in] height
- * The height of the \b input feature_map.
+ * The height of the input tensor \b feature_map.
  * @param[in] width
- * The width of the \b input feature_map.
+ * The width of the input tensor \b eature_map.
```
Comment (duzekunKTH) [inline bangc-ops/mlu_op.h:2459]:
```suggestion
 * The width of the input tensor \b feature_map.
```

File: bangc-ops/mlu_op.h (hunk 1 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:2246]: 已修改

File: bangc-ops/mlu_op.h (hunk 3 above)
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:2459]: 已修改

File: bangc-ops/mlu_op.h [hunk 4]
```diff
@@ -3938,7 +3929,7 @@ mluOpBallQuery(mluOpHandle_t handle,
  * @par Note
  * - You can specify the stride of all dimensions for input_desc and output_desc
  *   with ::mluOpSetTensorDescriptorEx.
- * - Data type of input tensor and output tensor must be the same.
+ * - Data types of input tensor and output tensor must be the same.
```
[the remark this answers is not in the document]
Comment (GraceGuanH) [reply bangc-ops/mlu_op.h:3933]: 改了

**Pull request merged**